### DISCUSSION TITLE
How to fix `jotai__WEBPACK_IMPORTED_MODULE_2__.useAtom) is not a function` when using jotai with next js

### DISCUSSION THREAD

**Discussion by kkatodus** [pmndrs/jotai#2595] [General] [answered]
I took forever to find a solution to the problem so I will share the solution here.

Add `"use client"` at the top of your file where you are trying to import the `useAtom` hook and you are good.

**Comment by DavidJFelix** [comment, marked as answer]
Thanks for sharing your solution. I'll look through docs and see if this can be surfaced more easily.
